build.js

```diff
@@ -14,13 +14,20 @@ import { esbuildStylesPlugins } from './pkg/lib/esbuild-common.js';
 import { eslintPlugin } from './pkg/lib/esbuild-eslint-plugin.js';
 import { stylelintPlugin } from './pkg/lib/esbuild-stylelint-plugin.js';
 
+const production = process.env.NODE_ENV === 'production';
 const useWasm = os.arch() !== 'x64';
 const esbuild = (await import(useWasm ? 'esbuild-wasm' : 'esbuild')).default;
 
-const production = process.env.NODE_ENV === 'production';
-const watchMode = process.env.ESBUILD_WATCH === "true";
-// linters dominate the build time, so disable them for production builds by default, but enable in watch mode
-const lint = process.env.LINT ? (process.env.LINT !== '0') : (watchMode || !production);
```

Comment:
I thought we supported this still in `cockpit` but just checked and it seems not!.

Reply:
But shouldn't we keep `LINT` to be flexible in Makefile's @martinpitt what do you think?